src/indices-to-html/index.js

```diff
@@ -26,14 +26,15 @@ function render_range(new_sub_text, new_sub_range, range_info, range_data, optio
         type_mappings = {
             b: 'strong', // be strong, my friend
             i: 'em', // I am, don't worry
+			noticon: 'gridicon',
```

Comment:
just noticed from this line that prettier hadn't appeared to run. we can just add a space